eng/scripts/tests/Invoke-GenerateAndBuildV2.tests.ps1

```diff
@@ -0,0 +1,52 @@
+#Requires -Version 7.0
+. (Join-Path $PSScriptRoot ".." ".." "common" "scripts" "Helpers" PSModule-Helpers.ps1)
+
+Install-ModuleIfNotInstalled "Pester" "5.3.3" | Import-Module
+
+Describe "Generate And Build SDK" -Tag "pipeline" {
+    BeforeAll {
+        # . $PSScriptRoot/../Invoke-GenerateAndBuildV2.ps1
+        # clean
+        $sdkRootPath =  (Join-Path $PSScriptRoot .. .. ..)
+        $sdkFolder = (Join-Path $sdkRootPath "sdk" "deviceupdate" "Azure.IoT.DeviceUpdate")
+        if (Test-Path $sdkFolder) {
+            Remove-Item $sdkFolder -Recurse
+        }
+        $artifactFolder = (Join-Path $sdkRootPath "artifacts" "packages" "Debug" "Azure.IoT.DeviceUpdate")
+        if (Test-Path $artifactFolder) {
+            Remove-Item $artifactFolder -Recurse
+        }
+    }
+
+    It "scenaro 1: first onboard" {
+        & $PSScriptRoot/../Invoke-GenerateAndBuildV2.ps1 -inputJsonFile $PSScriptRoot/firstOnboardInput.json -outputJsonFile $PSScriptRoot/firstOnboardOutput.json
+        $outputJson = Get-Content $PSScriptRoot/firstOnboardOutput.json | Out-String | ConvertFrom-Json
+
+        $outputJson.packages.Count | Should -Be 1
+        $outputJson.packages[0].packageName | Should -Be "Azure.IoT.DeviceUpdate"
+        $outputJson.packages[0].result | Should -Be "succeeded"
+        Remove-Item $PSScriptRoot/firstOnboardOutput.json
```

Comment:
You remove the output file here but not in the below tests. Was that intended?

Reply:
Yes, the output file is not used in below tests, so it is not needed.